twitter4j-ads/src/twitter4jads/internal/models4j/TwitterImpl.java

```diff
@@ -81,19 +82,19 @@ public TwitterImpl(Configuration conf, Authorization auth) {
         HttpParameter[] implicitParams = implicitParamsMap.get(conf);
         String implicitParamsStr = implicitParamsStrMap.get(conf);
         if (implicitParams == null) {
-            String includeEntities = conf.isIncludeEntitiesEnabled() ? "1" : "0";
-            String includeRTs = conf.isIncludeRTsEnabled() ? "1" : "0";
-            boolean contributorsEnabled = conf.getContributingTo() != -1L;
-            implicitParamsStr = "include_entities=" + includeEntities + "&include_rts=" + includeRTs +
-                                (contributorsEnabled ? "&contributingto=" + conf.getContributingTo() : "");
-            implicitParamsStrMap.put(conf, implicitParamsStr);
+//            String includeEntities = conf.isIncludeEntitiesEnabled() ? "1" : "0";
+//            String includeRTs = conf.isIncludeRTsEnabled() ? "1" : "0";
+//            boolean contributorsEnabled = conf.getContributingTo() != -1L;
+//            implicitParamsStr = "include_entities=" + includeEntities + "&include_rts=" + includeRTs +
+//                                (contributorsEnabled ? "&contributingto=" + conf.getContributingTo() : "");
```

Comment:
why are these code segments commented out?

Reply:
Sorry for that, I accidentally created this PR from my forks master branch. I'll send you separate PR's per Issue with acc. explanation. Thanks!